3p/integration.js

```diff
@@ -29,6 +29,7 @@ import {adsense} from '../ads/adsense';
 import {adtech} from '../ads/adtech';
```

Comment:
Please also add yourself to `_config.js`. `adPreconnect` if you only know the host. `adPrefetch` if you can know the full URL after all.